 hugegraph-seed Pulling 
 hugegraph Pulling 
 hugegraph Error failed to resolve reference "docker.io/hugegraph/hugegraph:1.2.1": docker.io/hugegraph/hugegraph:1.2.1: not found
 hugegraph-seed  Interrupted
Error response from daemon: failed to resolve reference "docker.io/hugegraph/hugegraph:1.2.1": docker.io/hugegraph/hugegraph:1.2.1: not found
